libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx

```diff
@@ -14,6 +14,8 @@ import { FormSearchInputConfig, FormSearchInputData } from "generic-view/models"
 export const SearchInput: APIFC<FormSearchInputData, FormSearchInputConfig> = ({
   data,
   config,
+  className,
+  style,
```

Comment:
you can change it to `...props` and `<Wrapper ...props>`. it will be easier to pass data- attributes in the future if needed

Reply:
Changed in [fdb9c25](https://github.com/mudita/mudita-center/pull/1980/commits/fdb9c25bc7dc60c0467729ad7e7d23388f26bb44)